Lib/test/_test_multiprocessing.py

```diff
@@ -2611,6 +2620,41 @@ def errback(exc):
         p.close()
         p.join()
 
+    def test_broken_process_pool1(self):
```

Comment:
Call it `test_broken_process_pool_map_async` instead, or something.

Reply:
Renamed to be more precise (and including map_async).